<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="TabWidget" name="tab_widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="cube_menu">
     <property name="title">
      <string>Load Orbital Data...</string>
     </property>
     <addaction name="load_cube_file_action"/>
     <addaction name="load_cube_online_action"/>
    </widget>
    <widget class="QMenu" name="export_menu">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="show_matplotlib"/>
     <addaction name="export_txt"/>
     <addaction name="export_hdf5"/>
    </widget>
    <widget class="QMenu" name="log_menu">
     <property name="title">
      <string>Open .log Files...</string>
     </property>
     <addaction name="log_file_action"/>
     <addaction name="mod_log_file_action"/>
    </widget>
    <widget class="QMenu" name="load_hdf5_menu">
     <property name="title">
      <string>Load Sliced Data...</string>
     </property>
     <addaction name="load_hdf5_action"/>
     <addaction name="load_sliced_from_binding_energy_action"/>
     <addaction name="load_sliced_from_photon_energy_action"/>
     <addaction name="load_sliced_from_cubefile_action"/>
    </widget>
    <addaction name="load_hdf5_menu"/>
    <addaction name="cube_menu"/>
    <addaction name="separator"/>
    <addaction name="save_project_action"/>
    <addaction name="load_project_action"/>
    <addaction name="export_menu"/>
    <addaction name="separator"/>
    <addaction name="log_menu"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>Tabs</string>
    </property>
    <addaction name="open_sim_tab_action"/>
    <addaction name="open_profile_tab_action"/>
    <addaction name="open_lmfit_tab_action"/>
    <addaction name="open_splitview_tab_action"/>
    <addaction name="separator"/>
    <addaction name="duplicate_tab_action"/>
    <addaction name="close_tab_action"/>
   </widget>
   <widget class="QMenu" name="settings_menu">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="general_menu">
     <property name="title">
      <string>General Settings</string>
     </property>
     <addaction name="general_action"/>
     <addaction name="general_default_action"/>
    </widget>
    <widget class="QMenu" name="logging_menu">
     <property name="title">
      <string>Logging Settings</string>
     </property>
     <addaction name="logging_action"/>
     <addaction name="logging_default_action"/>
    </widget>
    <widget class="QMenu" name="shortcut_menu">
     <property name="title">
      <string>Shortcut Settings</string>
     </property>
     <addaction name="shortcut_action"/>
     <addaction name="shortcut_default_action"/>
    </widget>
    <addaction name="general_menu"/>
    <addaction name="logging_menu"/>
    <addaction name="shortcut_menu"/>
    <addaction name="separator"/>
    <addaction name="settings_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="readme_action"/>
    <addaction name="welcome_action"/>
    <addaction name="about_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="settings_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="open_sim_tab_action">
   <property name="text">
    <string>Open new Orbital Tab</string>
   </property>
  </action>
  <action name="load_cube_file_action">
   <property name="text">
    <string>Locally...</string>
   </property>
  </action>
  <action name="load_cube_online_action">
   <property name="text">
    <string>Online...</string>
   </property>
  </action>
  <action name="show_matplotlib">
   <property name="text">
    <string>Matplotlib</string>
   </property>
  </action>
  <action name="log_file_action">
   <property name="text">
    <string>default.log...</string>
   </property>
  </action>
  <action name="mod_log_file_action">
   <property name="text">
    <string>module.log</string>
   </property>
  </action>
  <action name="general_action">
   <property name="text">
    <string>Edit User</string>
   </property>
  </action>
  <action name="general_default_action">
   <property name="text">
    <string>Show Default</string>
   </property>
  </action>
  <action name="logging_action">
   <property name="text">
    <string>Edit User</string>
   </property>
  </action>
  <action name="logging_default_action">
   <property name="text">
    <string>Show Default</string>
   </property>
  </action>
  <action name="settings_action">
   <property name="text">
    <string>Reload Settings</string>
   </property>
  </action>
  <action name="readme_action">
   <property name="text">
    <string>Open README</string>
   </property>
  </action>
  <action name="welcome_action">
   <property name="text">
    <string>Open Welcome</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About kMap.py</string>
   </property>
  </action>
  <action name="ref_action">
   <property name="text">
    <string>Print References</string>
   </property>
  </action>
  <action name="load_hdf5_action">
   <property name="text">
    <string>Locally...</string>
   </property>
  </action>
  <action name="load_sliced_from_binding_energy_action">
   <property name="text">
    <string>From Orbitals ...</string>
   </property>
  </action>
  <action name="load_sliced_from_photon_energy_action">
   <property name="text">
    <string>From hnu-dependence ...</string>
   </property>
  </action>
  <action name="open_profile_tab_action">
   <property name="text">
    <string>Open new Profile Tab</string>
   </property>
  </action>
  <action name="open_lmfit_tab_action">
   <property name="text">
    <string>Open new LM-Fit Tab</string>
   </property>
  </action>
  <action name="rename_current_tab_action">
   <property name="text">
    <string>Rename current Tab</string>
   </property>
  </action>
  <action name="export_txt">
   <property name="text">
    <string>Textfile</string>
   </property>
   <property name="toolTip">
    <string>Exports the contents of the current tab, if possible, as .txt file.

Currently the profile plot and the lmfit plot tab support this feature.</string>
   </property>
  </action>
  <action name="load_sliced_from_cubefile_action">
   <property name="text">
    <string>From Cubefile ...</string>
   </property>
  </action>
  <action name="duplicate_tab_action">
   <property name="text">
    <string>Duplicate current Tab</string>
   </property>
  </action>
  <action name="close_tab_action">
   <property name="text">
    <string>Close current Tab</string>
   </property>
  </action>
  <action name="save_project_action">
   <property name="text">
    <string>Save Project...</string>
   </property>
  </action>
  <action name="load_project_action">
   <property name="text">
    <string>Load Project...</string>
   </property>
  </action>
  <action name="export_hdf5">
   <property name="text">
    <string>Hdf5</string>
   </property>
  </action>
  <action name="open_splitview_tab_action">
   <property name="text">
    <string>Open new Split-View Tab</string>
   </property>
  </action>
  <action name="shortcut_action">
   <property name="text">
    <string>Edit User</string>
   </property>
  </action>
  <action name="shortcut_default_action">
   <property name="text">
    <string>Show Default</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QWidget</extends>
   <header>kmap/controller/tabwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
